### PR TITLE
Release v0.6.0

### PR DESCRIPTION
**Pull request by blkperl**
* [CHANGE] Update minimum Go version to 1.20 (#145)
* [FEATURE] Add TLS authentication using official exporter-toolkit (#113)
* [BUGFIX] Restore version command line flag and fix build info (#104)
* [FEATURE] Added s390x support to docker image (#98)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ## master / unreleased
 
+## 0.6.0 / 2023-12-08
```
Comment (blkperl) [inline CHANGELOG.md:3]: @SuperQ reminder to change this date before merge :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+
+* [CHANGE] Update minimum Go version to 1.20 (#145)
```
Comment (SuperQ) [inline CHANGELOG.md:5]: We don't typically mention non-user-facing changes like build stuff.

```suggestion
```

File: CHANGELOG.md (hunk 1 above)
Comment (SuperQ) [inline CHANGELOG.md:3]:
```suggestion
## 0.6.0 / 2023-12-09
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+* [FEATURE] Add TLS authentication using official exporter-toolkit (#113)
+* [BUGFIX] Restore version command line flag and fix build info (#104)
+* [FEATURE] Added s390x support to docker image (#98)
```
Comment (SuperQ) [inline CHANGELOG.md:8]: Usually we group things by category in the order of CHANGE, FEATURE, ENHANCEMENT, BUGFIX.

**Review by SuperQ — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (blkperl) [reply CHANGELOG.md:6]: Fixed!

**Review by SuperQ — APPROVED** [review]

**Pull request merged**